skbio/diversity/beta/_base.py

```diff
@@ -44,15 +47,25 @@ def pw_distances(counts, ids=None, metric="braycurtis"):
 
     See Also
     --------
+    unweighted_unifrac
+    weighted_unifrac
     scipy.spatial.distance.pdist
     pw_distances_from_table
 
     """
+    # this should probably go somewhere else. where?
+    _skbio_metrics = {'unweighted_unifrac': unweighted_unifrac,
```

Comment:
Probably at the global scope in this module or elsewhere, there isn't a benefit in constructing this lookup everytime the method is called